src/mds/Server.cc

```diff
@@ -5432,26 +5432,45 @@ int Server::parse_layout_vxattr(string name, string value, const OSDMap& osdmap,
 				file_layout_t *layout, bool validate)
 {
   dout(20) << "parse_layout_vxattr name " << name << " value '" << value << "'" << dendl;
+  auto parse_pool = [&](std::string& pool) -> int64_t {
+    try {
+      return boost::lexical_cast<int64_t>(pool);
+    } catch (boost::bad_lexical_cast const&) {
+      int64_t pool_id = osdmap.lookup_pg_pool_name(pool);
+      if (pool_id < 0) {
+	dout(10) << " unknown pool " << pool << dendl;
```

Comment:
nit: prefix function name using `__func__` for aid to debugging...

Reply:
added `__func__` to `dout()` for better debugging